custom_components/steam_wishlist/sensor_manager.py

```diff
@@ -113,8 +113,9 @@ class SensorManager:
     NOTE: This is intended to be a singleton.
     """
 
-    def __init__(self, hass: core.HomeAssistant, url: str) -> None:
+    def __init__(self, hass: core.HomeAssistant, config_entry, url: str) -> None:
         self.hass = hass
+        self.config_entry = config_entry
```

Comment:
Only pass the boolean instead of the config entry instance.

```suggestion
    def __init__(self, hass: core.HomeAssistant, store_all_wishlist_items: bool, url: str) -> None:
        self.hass = hass
        self.store_all_wishlist_items = store_all_wishlist_items
```